README.md

```diff
@@ -84,33 +84,28 @@ Relayer:
 | `RELAYER_NEUTRON_CHAIN_CONNECTION_ID`            | `string`          | neutron chain connection ID                                                                                                                                                | required |
-| `RELAYER_NEUTRON_CHAIN_CLIENT_ID `               | `string`          | IBC client ID for an IBC connection between Neutron chain and target chain (where the result was obtained from)                                                            | required |
 | `RELAYER_NEUTRON_CHAIN_DEBUG `                   | `bool`            | flag to run neutron chain provider in debug mode                                                                                                                           | required |
 | `RELAYER_NEUTRON_CHAIN_KEYRING_BACKEND`          | `string`          | [see](https://docs.cosmos.network/master/run-node/keyring.html#the-kwallet-backend)                                                                                        | required |
 | `RELAYER_NEUTRON_CHAIN_OUTPUT_FORMAT`            | `json`  OR `yaml` | neutron chain provider output format                                                                                                                                       | required |
 | `RELAYER_NEUTRON_CHAIN_SIGN_MODE_STR `           | `string`          | [see](https://docs.cosmos.network/master/core/transactions.html#signing-transactions) also consider use short variation, e.g. `direct`                                     | required |
 | `RELAYER_TARGET_CHAIN_RPC_ADDR`                  | `string`          | rpc address of target chain                                                                                                                                                | required |
-| `RELAYER_TARGET_CHAIN_CHAIN_ID `                 | `string`          | target chain id                                                                                                                                                            | required |
 | `RELAYER_TARGET_CHAIN_ACCOUNT_PREFIX `           | `string`          | target chain account prefix                                                                                                                                                | required |
 | `RELAYER_TARGET_CHAIN_VALIDATOR_ACCOUNT_PREFIX ` | `string`          | target chain validator account prefix                                                                                                                                      | required |
 | `RELAYER_TARGET_CHAIN_TIMEOUT `                  | `time`            | timeout of target chain provider                                                                                                                                           | required |
```

Comment:
You have removed this parameter from example config yet it is specified as required in README :thinking:

Reply:
Hm, added it back to .env.example